docs/features.md

```diff
@@ -0,0 +1,31 @@
+---
+Order: 20
+Title: Features
+Description: Features of the Cake.Issues.DupFinder addin.
+---
+The [Cake.Issues.DupFinder addin] provides the following features:
+
+# Basic features
+
+* Reads code duplicates from [JetBrains dupFinder] log files.
+
+# Supported IIssue properties
+
+|                                                                    | Property                          | Remarks                          |
+|--------------------------------------------------------------------|-----------------------------------|----------------------------------|
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.ProviderType`             |                                  |
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.ProviderName`             |                                  |
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.ProjectName`              |                                  |
```

Comment:
I think `ProjectName` is not set?

Reply:
Nope, I dont have any info regarding projects